packages/core/src/components/card/card.tsx

```diff
@@ -14,27 +14,11 @@ export class Card implements ComponentInterface {
   @Element()
   host!: HTMLDsoCardElement;
 
-  /**
-   * Whether or not the Card is clickable. This is NOT a boolean attribute. Set to "false" to make the Card non-clickable.
-   *
-   * @deprecated Use `href` instead and `<ELEMENT_TYPE slot="heading">` should NOT be of element type `a` (anchor).
-   */
-  @Prop({ reflect: true })
-  clickable = true;
-
-  /**
-   * Presentation of image in header.
-   *  - "normal" ("24 x 24").
-   *  - "wide" ("30 x 26")
-   */
-  @Prop({ reflect: true })
-  imageShape: ImageShape = "normal";
-
   /**
    * The URL to which the Card heading links.
    */
   @Prop({ reflect: true })
-  href?: string;
+  href!: string;
```

Comment:
De conventie is dat we Web Componenten altijd voorbereiden op incomplete informatie. Met de non-null assertion operator beweren wij dat we het beter weten dan de TypeScript compiler, maar wij kunnen geen garantie afgeven dat een afnemer altijd deze property zet.

Het is jammer dat Stencil deze operator heeft gekoppeld aan de notatie "required" in de gegenereerde documentatie.

Dit is iets anders dan de Component Models waarin wij wel een onderscheid hebben tussen optionele en verplichte properties. Dat komt omdat die interfaces een technische representatie zijn van de functionele wensen. Daarbij zorgt die strenge modellering ervoor dat wij in het documentatieplatform geen missers maken.

Reply:
Ik wil deze graag pair programmend aanpakken

Reply:
En we zijn tot de volgende constructie gekomen

```Typescript
href!: string | undefined;
```

Opdat we in de `readme.md` wel terugzien dat de property verplicht is, maar toch ook `undefined` kan zijn:

```md
| `href` _(required)_ | `href`    | The URL to which the Card heading links.                                          | `string \| undefined` | `undefined` |
```

Reply:
Echt mooi